userguide/virtualmachines.rst

```diff
@@ -201,7 +201,7 @@ a Virtual Machine (VM) type are described in
    |          |                    |                |                                                                                               |
    |          |                    |                |                                                                                               |
    +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
-   | 6        | Choose             | browse button  | Click |ui-browse| to select the installation media stored on the %brand% system.              |
+   | 6        | Optional: Choose   | browse button  | Click |ui-browse| to select an installation media stored on the %brand% system.               |
```

Comment:
Not clear what "an installation media stored" means.  Better to say "an installer ISO or image file on" and leave out "stored".